doc/README.CTSM

```diff
@@ -0,0 +1,25 @@
+=====================
+  Using CTSM with WRF
+=====================
+
+CTSM has now been coupled to WRF. 
+CTSM can be used as the land surface model scheme in WRF by choosing sf_surface_physics = 6
+in the WRF namelist.input file.
+
+For comprehensive instructions on how to run WRF with CTSM please check: 
+https://escomp.github.io/ctsm-docs/versions/master/html/lilac/specific-atm-models/wrf.html
+
+In summary, he procedure for running WRF-CTSM is similar to the workflow for 
```

Comment:
Typo here: it should be 'the', not 'he'.
Also, perhaps the info on CTSM can be added here (e.g. the GitHub location).

Reply:
Thanks for your comment. I just corrected the typo.